packages/@sanity/dashboard/src/components/dashboardWidget.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import {Card, Box, Heading} from '@sanity/ui'
+import styled from 'styled-components'
+
+const Root = styled(Card)`
+  overflow: auto;
+`
+
+const Header = styled(Card)`
+  position: sticky;
+  top: 0;
+  z-index: 2;
+`
+
+const Footer = styled(Card)`
+  position: sticky;
+  bottom: 0;
+  z-index: 2;
+`
+
+const Content = styled(Box)`
+  position: relative;
+  z-index: 1;
+  overflow: auto;
+  outline: none;
+`
+
+interface DashboardWidgetProps {
+  header?: string
+  children: React.ReactNode
+  footer?: React.ReactNode
```

Comment:
Pondering if we should extend this widget component to allow for "actions" or some such - eg a way to put consistent-looking buttons/links in the footer. Let's leave it for now and revisit later, though

Reply:
Yes, I thought about it too. However I like that the component provides 100% flexibility for what can be rendered in the footer. But I agree with you, let's revisit later!